README.md

```diff
@@ -844,8 +844,6 @@ Services to securely store your Docker images.
 
 ## Contributor over time
 
-[![Contributor over time](https://api7.ai/contributor-graph?chart=contributorOverTime&repo=veggiemonk/awesome-docker)
```

Comment:
@veggiemonk should we remove this graph or is there a possibility to fix the URL instead?